rdbms/src/main/java/org/spine3/server/storage/jdbc/command/query/UpdateCommandQuery.java

```diff
@@ -74,4 +74,17 @@ protected Builder getThis() {
             return this;
         }
     }
+
+    private enum QueryParameter {
+
+        RECORD(1),
+        STATUS(2),
+        ID(3);
+
+        private final int index;
```

Comment:
No, it cannot. The value is used as a constant to store the indexes of the query params.